.gitignore

```diff
@@ -35,6 +35,7 @@ __pycache__/
 
 # C extensions
 *.so
+*.c
```

Comment:
I see why you added this, but maybe it would be better to add another `.gitignore` in the `weighted_levenshtein/` directory with just the line `clev.c`.

This `.gitignore` is automatically generated via gitignore.io so that one line change could be easily missed in the future if someone were to regenerate the file, plus ignoring all `.c` files might not be what we want in the future so I think this would be clearer.